### PR TITLE
Add creation static page feature

### PR DESCRIPTION
**Pull request by tony-tran**

### CODE REVIEW

File: app/migrations/Version20171227163801.php [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+namespace Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+class Version20171227163801 extends AbstractMigration
+{
+    public function up(Schema $schema)
+    {
+        $this->addSql('UPDATE pages SET slug=\'emmanuel-macron\' WHERE slug= \'emmanuel-macron-ce-que-je-suis\'');
```
Comment (HeahDude) [inline app/migrations/Version20171227163801.php]: You must also handle 301s. See https://github.com/EnMarche/en-marche.fr/blob/281f460f0f5425c27787642d0bd0426381006120/src/Redirection/LegacyRedirectionsSubscriber.php#L35.

Comment (tony-tran) [reply app/migrations/Version20171227163801.php]: I don't need to do redirection, I change the slug for the `findOneBySlug`. Look changes on `PageController.php`

Comment (B-Galati) [reply app/migrations/Version20171227163801.php]: Why did you change all the slugs in that case ? I think I am missing something..

File: app/config/routing.yml [hunk 2]
```diff
@@ -96,3 +96,12 @@ app_redirecting:
     host:     "%app_host%"
     resource: "@AppBundle/Controller/RedirectingController.php"
     type:     "annotation"
+
+# This route must be loaded the last to prevent an override
+app_static_page:
+    host:    "%app_host%"
+    path:    /s/{slug}
```
Comment (B-Galati) [inline app/config/routing.yml]: We really want this weird path ? Looks bad for SEO no ?

File: app/migrations/Version20171227163801.php (hunk 1 above)
Comment (B-Galati) [reply app/migrations/Version20171227163801.php]: It is to simplify the logic inside `SitemapFactory.php`, that's it ?

Comment (tony-tran) [reply app/migrations/Version20171227163801.php]: I change all slugs because they are all wrong in database. And I need to change slug to generate the right URL in `SitemapFactory.php`.

File: app/config/routing.yml (hunk 2 above)
Comment (tony-tran) [reply app/config/routing.yml]: There is already a route with `/{url}` for path. To prevent some errors I prefer use this url and it is write in the ticket of this feature ;)

File: app/migrations/Version20171227163801.php (hunk 1 above)
Comment (HeahDude) [reply app/migrations/Version20171227163801.php]: I don't understand your answer, the Uris did change right?

Comment (HeahDude) [reply app/migrations/Version20171227163801.php]: I mean a changed uri should now respond a 404?

Comment (HeahDude) [reply app/migrations/Version20171227163801.php]: If "yes" is the answer to my questions above, you MUST handle 301.

Comment (B-Galati) [reply app/migrations/Version20171227163801.php]: No URLs are still the same ;-)

Comment (HeahDude) [reply app/migrations/Version20171227163801.php]: Ok, got it, thx!
